### PR TITLE
[NIS-13915] Implemented authorization API presentation layer

### PR DESCRIPTION
**Pull request by westcrime**

### CODE REVIEW

File: Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        public static IServiceCollection AddPresentationDependencyInjection(this IServiceCollection services, IConfiguration configuration)
+        {
+            services.AddProblemDetails();
+
+            services.AddApplicationDependencyInjection(configuration);
+
+            services.AddRazorPages();
+
+            var connectionString = configuration.GetConnectionString("DefaultConnection");
+            var isBuilder = services
```
Comment (Glipchik) [inline Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs:22]: What is isBuilder for?

File: Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                new IdentityResource("email", new[] { JwtClaimTypes.Email }),
+                new IdentityResource("roles", new[] { JwtClaimTypes.Role })
+            };
+
+            foreach (var resource in identityResources)
+            {
+                if (!context.IdentityResources.Any(ir => ir.Name == resource.Name))
+                {
+                    context.IdentityResources.Add(resource.ToEntity());
+                    Console.WriteLine($"Added Identity Resource: {resource.Name}");
```
Comment (Glipchik) [inline Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs:119]: Either change it to writing to logs or remove it

File: Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                new ApiScope("api_email", "Access to email", new[] { JwtClaimTypes.Email }),
+                new ApiScope("api_roles", "Access to roles", new[] { JwtClaimTypes.Role })
+            };
+
+            foreach (var scope in scopes)
+            {
+                if (!context.ApiScopes.Any(s => s.Name == scope.Name))
+                {
+                    context.ApiScopes.Add(scope.ToEntity());
+                    Console.WriteLine($"Added API scope: {scope.Name}");
```
Comment (Glipchik) [inline Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs:141]: Same

**Review by Glipchik — CHANGES_REQUESTED** [review]

File: Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs (hunk 1 above)
Comment (westcrime) [reply Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs:22]: It was autogenerated, it is useless, removed

File: Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs (hunk 2 above)
Comment (westcrime) [reply Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs:119]: Fixed

File: Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs (hunk 3 above)
Comment (westcrime) [reply Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs:141]: Fixed

**Review by Glipchik — APPROVED** [review]

**Pull request merged**